src/vaadin-grid-sorter.html

```diff
@@ -25,6 +25,7 @@
       :host {
         display: inline-flex;
         cursor: pointer;
+        max-width: 100%;
```

Comment:
Couldn't get it working nice with just this fix. `[part="content"]` also needs 100% (max) width?

Reply:
You’re right, thanks! I’ll fix it.

Reply:
That hopefully does it.